scripts/set-rn-version.js

```diff
@@ -47,8 +52,7 @@ let major,
 try {
   ({major, minor, patch, prerelease} = parseVersion(version, buildType));
 } catch (e) {
-  echo(e.message);
-  exit(1);
+  throw e;
```

Comment:
What's the exit code of the script when it throws? I'm asking because I think that we are calling `set-rn-version` directly also in CI, so we have to make sure that it stopped the job in that case as well (pretty sure that that's what is going to happen)

Reply:
it's always 1 AFAIK so it will stop the job there too if it's correctly addressed. The issue with exec is that it tends to silence the errors which is why I had to add the extra bit of logic of checking the exitCode separately.

Reply:
Same as above. If something throws and is not in a try-catch, the exception should bubble, shouldn't it?